lib/src/model/model.dart

```diff
@@ -1,8 +1,7 @@
 import 'dart:collection';
-
 import 'package:casbin/src/config/config.dart';
-import 'package:casbin/src/model/assertion.dart';
-import 'package:casbin/src/model/policy.dart';
+import '../rbac/role_manager.dart';
+import 'policy.dart';
```

Comment:
And this is probably best to leave as it is. Since `policy.dart` is in the same dir.